contents/code/tile.js

```diff
@@ -112,6 +112,7 @@ function Tile(firstClient, tileIndex) {
             firstClient.noBorder = true;
         }
         this.maximized = false;
+        this._canSetMaximize = (firstClient.setMaximize != null);
```

Comment:
What's the use in saving this?

It doesn't check if _all_ clients can do it, I don't think it's costly to just check again, I'm not sure if it can change?

Reply:
I do not know what's the cost of checking for the presence of a native method (perhaps negligible?) and since it cannot change, I opted for caching.

Reply:
I missed the first part of the question. No, it definitely can not change in runtime. As I said above, I opted for caching, because I'm sure it cannot change and I don't know the cost of the test.